Issue/gui2535 missing placeholder

chrisgarrity opened the pull request:
### Resolves

_What Github issue does this resolve (please include link)?_
Fixes the second part of https://github.com/LLK/scratch-gui/issues/2535

### Proposed Changes

_Describe what this Pull Request does_
Localizes the `block name` default text when creating a custom procedure.
The second commit just updates translations and doesn't need review.

### Reason for Changes

_Explain why these changes should be made_
localization

/cc @AnmAtAnm

chrisgarrity committed:
Localize custom procedure default name

chrisgarrity committed:
update translations

AnmAtAnm commented on core/procedures.js:
```diff
@@ -374,7 +374,7 @@ Blockly.Procedures.getPrototypeBlock = function(procCode, workspace) {
   var mutationText = '<xml>' +
       '<mutation' +
-      ' proccode="block name"' +
+      ' proccode="' + Blockly.Msg.PROCEDURE_DEFAULT_NAME + '"' +
```
To avoid renames with Closure advanced compilation, prefer `Blockly.Msg['PROCEDURE_DEFAULT_NAME']`, unless you're including translation in your compilation sources.

AnmAtAnm commented:
FWIW, this is comparable to `Blockly.Msg['PROCEDURES_DEFNORETURN_PROCEDURE']` in main Blockly.

chrisgarrity committed:
Addressing comment

From @AnmAtAnm: To avoid renames with Closure advanced compilation, prefer Blockly.Msg['PROCEDURE_DEFAULT_NAME'], unless you're including translation in your compilation sources.

Also resync translations to fix conflicts.
